packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -125,6 +125,66 @@ export const DeckLayout = ({
           onComplementarySidebarOpenChange: (next) => (context.complementarySidebarOpen = next),
         })}
       >
+        {/* Dialog overlay to dismiss dialogs. */}
+        <Main.Overlay />
+
+        {/* Nav sidebar */}
+        <Sidebar attention={attention} layoutParts={layoutParts} />
+
+        {/* Main content area */}
+        {isEmpty ? (
+          <Main.Content handlesFocus>
+            <ContentEmpty />
+          </Main.Content>
+        ) : (
+          <Main.Content bounce classNames='grid block-end-[--statusbar-size]' handlesFocus>
+            <div role='none' className={layoutMode === 'solo' ? 'contents' : 'relative'}>
+              <Deck.Root
+                classNames={[
+                  !flatDeck && 'surface-deck',
+                  layoutMode === 'deck' && [
+                    'absolute inset-0',
+                    'transition-[padding] duration-200 ease-in-out',
+                    slots?.wallpaper?.classNames,
+                  ],
+                ]}
+                solo={layoutMode === 'solo'}
+                style={{ ...overscrollAmount }}
+              >
+                {layoutMode === 'solo' &&
```

Comment:
changed a lot of this in
https://github.com/dxos/dxos/pull/7642

Reply:
I noticed, I rearranged these initially so their DOM order would allow tabbing, but I ended up needing it to do something different and now the DOM order doesn’t matter, so this should be easy enough to de-conflict.